stream.go

```diff
@@ -300,7 +307,21 @@ func (ss *serverStream) SetTrailer(md metadata.MD) {
 	return
 }
 
-func (ss *serverStream) SendMsg(m interface{}) error {
+func (ss *serverStream) SendMsg(m interface{}) (err error) {
+	defer func() {
+		if ss.tracing {
+			ss.mu.Lock()
+			if ss.traceInfo.tr != nil {
```

Comment:
if `ss.tracing` is true, this should always be true.

Or is there a way that SendMsg can get called after the stream is finished?